.azure-pipelines/pipelines.yml

```diff
@@ -183,6 +183,15 @@ jobs:
           testRunTitle: "macOS"
         condition: always()
 
+      - script: ./tools/flaky_test/process_xml.py
```

Comment:
I think you can use the shell script to do this, then you don't need another pip install in mac_ci_steps.sh, doesn't it work?

Reply:
Yes. Will do.